.github/CONTRIBUTING.md

```diff
@@ -34,3 +35,8 @@ and help.
 Please see the [Developers' Guide](https://github.com/ethereum/go-ethereum/wiki/Developers'-Guide)
 for more details on configuring your environment, managing project dependencies
 and testing procedures.
+
+## Issue reproductibility
+
+Before you create an issue, we want you to try to reproduce it over the [quorum quick dev start framework](https://github.com/ConsenSys/quorum-dev-quickstart) with the latest release.
```

Comment:
Before you create an issue, please try to reproduce it with [quorum quick dev start framework](https://github.com/ConsenSys/quorum-dev-quickstart) using the latest release.